gst/nnstreamer/nnstreamer_plugin_api_util_impl.c

```diff
@@ -1009,6 +1005,27 @@ gst_tensor_dimension_is_valid (const tensor_dim dim)
   return TRUE;
 }
 
+/**
+ * @brief Compare the tensor dimension.
+ * @return TRUE if given tensors have same dimension.
+ */
+gboolean
+gst_tensor_dimension_is_equal (const tensor_dim dim1, const tensor_dim dim2)
+{
+  guint i;
+
+  /* Do not compare zero dimension. */
+  if (dim1[0] == 0U && dim2[0] == 0U)
```

Comment:
Instead of the operator '&&', maybe 'II'?

Reply:
updated